doc/getting-started/rt_industry_ubuntu.rst

```diff
@@ -405,6 +423,15 @@ Update the Grub file
 
       $ sudo update-grub
 
+#. Reboot into the Ubuntu Service VM
+
+Launch the RTVM
+===============
+
+  .. code-block::
```

Comment:
build failed because the code-block needs a language ``.. code-block:: none``  
I pushed a fix to your branch @gvancuts

Reply:
Thanks (and sorry for forgetting to test the `make doc` step...)